pkg/kubelet/container_bridge.go

```diff
@@ -121,13 +121,13 @@ func cbr0CidrCorrect(wantCIDR *net.IPNet) bool {
 // TODO(dawnchen): Using pkg/util/iptables
 func ensureIPTablesMasqRule() error {
 	// Check if the MASQUERADE rule exist or not
-	if err := exec.Command("iptables", "-t", "nat", "-C", "POSTROUTING", "-o", "eth0", "-j", "MASQUERADE", "!", "-d", "10.0.0.0/8").Run(); err == nil {
+	if err := exec.Command("iptables", "-t", "nat", "-C", "POSTROUTING", "-o", "eth0", "-j", "MASQUERADE", "!", "-d", "10.0.0.0/8", "-w").Run(); err == nil {
```

Comment:
Unfortunately some older implementations of iptables don't support -w, and we DO support those still.  I'm a bit wary of breaking back-compat with old iptables just yet (though it will happen at some point).

@ghodss for his thoughts, since he'll be the victim when we do break it.  I was thinking that 1.1 would stay compatible and 1.2 might be the breaker.